dashboard/app/controllers/courses_controller.rb

```diff
@@ -20,4 +29,21 @@ def show
 
     render 'show', locals: {course: course}
   end
+
+  def new
+    render 'new'
+  end
```

Comment:
Nit: We should standardize on whether we want to include these boilerplate action definitions if they are only rendering the default view.  I know you prefer explicit, but Rails is CoC by default 😛.